pkg/steps/template.go

```diff
@@ -620,9 +623,6 @@ func waitForPodCompletionOrTimeout(podClient coreclientset.PodInterface, name st
 				return false, nil
 			}
 			if podJobIsFailed(pod) {
-				if err := outputAnnotatedContainerLogs(podClient, pod); err != nil {
-					log.Printf("%v", err)
-				}
```

Comment:
Why is the output being removed here? If we miss the first event, removing this would mean we do not log the failed container output, woudn't it?

Reply:
It will.  See https://github.com/openshift/ci-operator/pull/193/files#diff-37ed7e4b3e5f7bb5d23118bff1f75bd9R597

Reply:
That happens before this removed code, and may not hit (otherwise this cycle would not be needed).